web/pgadmin/tools/sqleditor/tests/test_download_csv_query_tool.py

```
@@ -240,3 +243,282 @@ def tearDown(self):
+                 expected_content_type='application/json',
+                 expected_extension='.json', sql=AWKWARD_SQL,
+                 awkward_data=True)
+        ),
+        (
+            'Download awkward data as XML stays well formed',
+            dict(data_format='xml', add_bom=False, encoding='utf-8',
+                 expected_content_type='application/xml',
+                 expected_extension='.xml', sql=AWKWARD_SQL,
+                 awkward_data=True)
+        ),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)